network-api/networkapi/wagtailpages/templates/wagtailpages/mini_site_name_space.html

```diff
@@ -4,6 +4,8 @@
 
 {% block minisite_title_mb %}
   {% if uses_menu %}
+  {% block bannered_campaign_page_menu %}
```

Comment:
can we call this something that "makes sense" from the mini site's perspective, which knows nothing about which pages extend from it? E.g. `minisite_title_menu`?

Reply:
Sure. Btw, what does `minisite_title` mean? Is it the "title" section on minisite?